stingray/events.py

```diff
@@ -302,6 +285,25 @@ def read(self, filename, format_='pickle'):
         elif format_ == 'pickle':
             self = object
 
+        elif format_ == "fits":
+            ret = io.load_events_and_gtis(filename, **kwargs)
+            times = ret.ev_list
+            gti = ret.gti_list
+            
+            keys = ret.additional_data.keys()
+            values = []
+            attributes = ['energies', 'ncounts', 'mjdref', 'dt', 'notes', 'pi']
+
+            for attribute in attributes:
+                if attribute in keys:
+                    values.append(object[attribute])
+
+                else:
+                    values.append(None)
+
+            self = EventList(time=times, gti=gti, energies=values[0], ncounts=values[1],
+                mjdref=values[2], dt=values[3], notes=values[4], pi=values[5])
+
```

Comment:
@matteobachetti Please take a look at this implementation and let me know if it looks okay.